Makefile

```diff
@@ -33,8 +33,10 @@ override LDFLAGS += \
 # docker image publishing options
 DOCKER_PUSH=false
 IMAGE_TAG=latest
+PLUGIN_IMAGE_TAG=latest
 ifneq (${GIT_TAG},)
 IMAGE_TAG=${GIT_TAG}
+PLUGIN_IMAGE_TAG=${GIT_TAG}
```

Comment:
Actually I don't think these will ever be different than the controller version, so lets just reuse IMAGE_TAG and PLUGIN_IMAGE_TAG